Training/PythonWebServer/resources/stylesheet.css

```diff
@@ -1,5 +1,136 @@
-h1 {
+p{
+	display: inline;
+}
+html{
+	height: 100%;
+}
+body{
+	height: 100%;
+	margin: 0;
+	background-repeat: no-repeat;
+	background: linear-gradient(grey, white);
+	background-attachment: fixed;
+}
+.wrapper{
+	display: block;
+	margin-right: auto;
+	margin-left: auto;
+	width: 80%;
+	text-align: center;
+}
+.button{
+	box-shadow: 7px 7px grey;
+	border-radius: 5px;
+
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 4cm;
+	width: 330px;
+	height: 70px;
+
+	font-family: sans-serif;
+	text-align: center;
+	color: white;
+	font-weight: bold;
+	
+	background-color: #F5D04C;
+    border: none;
+    padding: 10px 75px;
+    font-size: 40px;
+    display: block;
+}
+button::-moz-focus-inner {
+  border: 0;
+}
+@media screen and (max-width: 1400px){
+.display{
+	box-shadow: 7px 7px grey;
+	border-radius: 5px;
+	display:inline-block;
+	letter-spacing: -2px;
+
+	margin-top: 3cm;
```

Comment:
Generally `cm` should be avoided, because it comes off very different on different screens.

In order of preference, I use `em, rem, px, vh`. They all have different purposes (you can read about them yourself) but in general that is the preference because that is the order in how nicely they play on different screen sizes.